### PR TITLE
Added rustup update to pre-commit scripts.

### PR DESCRIPTION
**Pull request by efrisch**
## Description

Follow-up pyrsia/pyrsia#592 adds `rustup update` to pre-commit scripts.

This PR adds running rustup update to the pre-commit scripts.

## Screenshots (optional)
Screen Shot 2022-04-18 at 2 16 31 PM

## PR Checklist

- [X] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [X] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/good_pr.md)
- [X] I've included a good title and brief description along with how to review them.
- [X] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).
- [X] I've requested a review  from `pyrsia/collaborators`.

### Code Contributions

- [X] I've built the code `cargo build --all-targets` successfully.
- [X] I've run the unit tests `cargo test --workspace` and everything passes.
- [X] I've made sure my rust toolchain is current `rustup update`.

### CODE REVIEW

**Review by prince-chrismc — APPROVED** [review]
LGTM!

**Review by fishseabowl — APPROVED** [review]
LGTM 🚀

File: pre-commit.bat [hunk 1]
```diff
@@ -13,6 +13,8 @@ cargo audit
 IF %ERRORLEVEL% NEQ 0 (ECHO Cargo audit failed. &Exit /b 1)
 cargo clippy
 IF %ERRORLEVEL% NEQ 0 (ECHO Cargo clippy failed. &Exit /b 1)
+rustup update
```
Comment (chb0github) [inline pre-commit.bat:16]: Does anyone still use `.bat` files? Powershell or bust

**Review by chb0github — APPROVED** [review]

File: pre-commit.bat (hunk 1 above)
Comment (efrisch) [reply pre-commit.bat:16]: I _think_ everyone is actually using WSL and the bat file (or powershell) are unnecessary. May drop an informal survey in slack today.

**Pull request merged**